website/docs/job.md

```diff
@@ -36,9 +36,38 @@ def schedule(self, initial_delay: str, frequency: str, job: Job):
 def schedule_once(self, initial_delay: str, job: Job):
     ... 
 ````

```

Comment:
please pull from massive-update. i added a log message upon job fail. 

add tests 
- run using `scheduler.schedule`  and see tests are running again and again 
- make a test useing `scheduler.schedule` when the job is failing, will it be runninfg again. 
- make a test useing `scheduler.schedule`, job will take 10 seconds to finish, 
   while will it is scheduled to run every 5 seconds. 
   what will happen then?  
   if it runs it is already running log a message that say already running. 
   make sure it is not running again.